mealie/schema/settings.py

```diff
@@ -15,18 +15,6 @@ class SiteSettings(CamelModel):
     class Config:
         orm_mode = True
 
-        schema_extra = {
```

Comment:
Why did you remove these?

Reply:
Those are now part of the seed data in alembic migrations

Reply:
> Those are now part of the seed data in alembic migrations

These are for providing documentation for the API. 

https://fastapi.tiangolo.com/tutorial/schema-extra-example/